### ISSUE TITLE
Ordering products in a category numerically not working

### ISSUE THREAD

**Issue by cavemanwayne** [dweeves/magmi-git#279]
I've tried ordering my products within their respective categories using '::NUMBER' after the category names but this info doesn't import to magento through Magmi.

**Comment by dweeves** [comment]
indeed, this ordering seems to currently work only with category_ids, since with names, separators are used for specifying category attributes values.

I'll try to fix this, probably by setting a specific separator for position.

**Comment by cavemanwayne** [comment]
This is great news please update me when the fix has been created and i'll update my spreadsheet accordingly

Regards Wayne

**Comment by cavemanwayne** [comment]
Hi dweeves has there been any progress on the ordering of categories?

**Comment by OferBarkai** [comment]
Joining the conversation, I would like to say first how awesome magmi is and that your work is priceless. I am using magmi for a year now and I cannot imagine working without it. So thank you dweeves.
Sorting products Is imperative in my shop, and dividing my csv file to avoid the sorting reset is messy. 
To have the option to sort with categories column (names) sound very very good. Truth is I did not manage to sort using category_ids as well for a reason unknown.
Thank you Dweeves.

**Comment by hgpit** [comment]
I would also love to this amendment. As it stands though, I can't even get the "category name::[is_active]::[is_anchor]::[include_in_menu]" approach to work. When I check my import, Magento has ignored the additional options. So I'm hoping that if Dweeves can find time to add the sort option it will also fix this :)

**Comment by n0kit31** [comment]
I'm looking forward for this feature!
